src/Components/Loading.js

```diff
@@ -3,7 +3,7 @@ import Logo from "../images/sachar.jpg";
 import '../css/app.css';
 
 const Loading=()=>{
-    return (
+    return (//this component is the main page component, it uses the css file to center the message and the image.
```

Comment:
Consider splitting the CSS part to a different comment

Reply:
i didn't understand the comment,
the css is in a different file.